processor/bandwidthBar/bandwidthBar.go

```diff
@@ -246,7 +248,9 @@ func (p *Processor) ProcessData(processorArgs structure.ProcessingFuncArgs, ack
 
 func main(){
 
-	addr, err := net.ResolveTCPAddr("tcp", "0.0.0.0:42586")
+	processorConfig := function.GetProcessorConfig("../../config/processor.json")
+
+	addr, err := net.ResolveTCPAddr("tcp", "0.0.0.0:"+processorConfig.ServerPort)
```

Comment:
hardcodes bindhost - may not always be advisable. earmark for future change to config.